.github/workflows/ci.yml

```
@@ -8,6 +8,46 @@ on:
 
 jobs:
 
+  bandit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-python@v1
+      - uses: actions/checkout@v2
+      - run: python -m pip install --upgrade bandit
+      - run: bandit -r hijack -x hijack/tests
+
+  black:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-python@v1
+      - uses: actions/checkout@v2
+      - run: python -m pip install black
+      - run: black --check --diff .
+
+  flake8:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-python@v1
+      - uses: actions/checkout@v2
+      - run: python -m pip install flake8
+      - run: flake8 .
+
+  isort:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-python@v1
+      - uses: actions/checkout@v2
+      - run: python -m pip install isort
+      - run: isort --check-only --diff --recursive .
```

--recursive option has been removed. Directories passed in are now automatically sorted recursive.
```suggestion
      - run: isort --check-only --diff  .
```